pkgs/tools/networking/tuntox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, libtoxcore, libsodium, libevent }:
+
+stdenv.mkDerivation rec {
+  name = "tuntox-${version}";
```

Comment:
Please replace with `pname = "tuntox";` and then you can drop `name` as it will be implied.